src/ToDoList.css

```
@@ -38,4 +40,72 @@ body {
     margin-left: 1620px;
     margin-top: 870px;
 }
-
+.newTaskBox {
```

```suggestion
.new-task-box {
```
Change it everywhere